src/main/services/CaseService.ts

```diff
@@ -67,3 +70,61 @@ export const getPreloginCaseData = (redisClient: RedisClient, guid: string): Pro
     });
   });
 };
+
+export const formatCaseData = (fromApiCaseData: initiateCaseDraftResponse): CaseWithId => ({
+  id: fromApiCaseData.id,
+  state: fromApiCaseData.state,
+  isASingleClaim: fromApiCaseData.case_data.case_type === 'Single' ? YesOrNo.YES : YesOrNo.NO,
+});
+
+export class CaseApi {
+  constructor(private readonly axio: AxiosInstance) {}
+
+  public async getCase(caseTypeId = CcdDataModel.SINGLE_CASE_ENGLAND): Promise<CaseWithId | false> {
+    try {
+      const fetchedCases = await this.getDraftCases(caseTypeId);
+      switch (fetchedCases.length) {
+        case 0: {
+          return false;
+        }
+        default: {
+          return formatCaseData(fetchedCases[fetchedCases.length - 1]);
+        }
+      }
+    } catch (err) {
+      throw new Error('Case could not be retrieved.');
```

Comment:
Looks like that the error caught needs to be handled, otherwise the app would just crashed by unexpected error.

Reply:
Ok I change these to use the logger for now, instead of throwing errors.